en/install.rst

```diff
@@ -6,13 +6,17 @@ Prerequisites
 -------------
 To build a PHP extension and use Zephir you need the following requirements:
 
+* `Zephir parser`_ >= 1.1.0
 * gcc >= 4.x/clang >= 3.x
 * re2c 0.13 or later
 * gnu make 3.81 or later
 * autoconf 2.31 or later
 * automake 1.14 or later
 * libpcre3
 * php development headers and tools
+* The build-essential package when using gcc on Ubuntu (and likely other distros as well)
+
+.. _Zephir parser: https://github.com/phalcon/php-zephir-parser
 
 If you're using Ubuntu, you can install the required packages this way:
```

Comment:
Should probably add `build-essential` to the example command, here, too...

Reply:
Cannot add code tag in .rst markup file

Reply:
I don't follow...

Reply:
All requests except this are done, coz don't know how to do it

Reply:
OK, so it looks like GitHub moved which lines I was commenting on, it a couple of cases. I'll see if I can change them...

Reply:
Line 24 in the current version contains a command to install needed packages. Adding `build-essential` to that command, as well, is what I was requesting, here. Not sure why GitHub thought I meant a different line number.

Reply:
Ok done